src/components/utils/jwt.js

```diff
@@ -22,3 +22,15 @@ export const getUrlsFromJWT = (token) => {
   }
   return urls
 }
+
+export const getEnterpriseFeaturesFromJWT = (token) => {
+  let enterpriseFeatures = {}
+  try {
+    const jwt = parseJwt(token)
+    enterpriseFeatures = jwt.enterprise_features
```

Comment:
Should just have `return jwt.enterprise_features` here rather than outside of the try catch statement. Also removes the need to define an empty `enterpriseFeatures` object at the start.

Reply:
Thanks, forgot to change this